examples/cuda/Makefile

```diff
@@ -25,25 +25,41 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# This Makefile fragment defines all of the regression tests (and the
-# source path) for this sub-directory.
+# This (rather sparse) Makefile is responsible for building and
+# running regression tests on F1.  Run `make help` to see the
+# available targets. (The target `help` is defined in f1.mk)
 
-REGRESSION_TESTS_TYPE = cuda
-SRC_PATH=$(REGRESSION_PATH)/$(REGRESSION_TESTS_TYPE)/
+# environment.mk verifies the build environment and sets the following
+# makefile variables:
+#
```

Comment:
This file is an amalgamation/move of the files tests.mk, and rules.mk and Makefile in regression/cuda/